src/openms/source/SYSTEM/UpdateCheck.cpp

```diff
@@ -167,6 +167,7 @@ namespace OpenMS
           }
         }
         delete query;
+        event_loop.quit();
```

Comment:
what about htis one?

Reply:
we always had a thread reserved for the update check

Reply:
idling but annoying